docs/kubernetes.md

```diff
@@ -25,6 +25,27 @@ To set up Faasm on [GKE](https://console.cloud.google.com/kubernetes) you can do
 - Check things are working by running `kubectl get nodes`
 - Install Knative serving as described below
 
+### Azure Kubernetes Service
+
+To set up Faasm on AKS, you can do the following:
+- Set up an account and install the [`az`](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) client.
+- Create a resource group with sufficient quota.
+- Run:
+```bash
+az aks create \
+    --resource-group <YOUR_RESOURCE_GROUP> \
+    --name <CLUSTER_NAME> \
+    --node-count <CLUSTER_NODE_COUNT> \
+    --node-vm-size <VM_SIZE> \
+    --generate-ssh-keys
+```
+- Lastly, to update your local config run:
+```bash
+az aks get-credentials \
+    --resource-group <YOUR_RESOURCE_GROUP> \
+    --name <CLUSTER_NAME>
+```
```

Comment:
Let's avoid specifying how people create a cluster (as with the GKE instructions above), you can like to the AKS docs if you think they're useful, but really all we care about is that someone has a cluster where they can run `kubectl`. 

You can add any recommended resource levels like in the GKE instructions above (e.g. use at least 4 hosts and nothing smaller than XYZ VMs)

Reply:
If you want to keep a specific script for the faasm experiments, you could create one in the `experiments-base` repo, e.g. `bin/provision_aks_cluster.sh`